website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,71 +1,87 @@
-Figma is an incredibly versatile tool, but this versatility can sometimes backfire leading to disorganization, inconsistency, and difficulty in handoffs and collaboration. Use these best practices to better organize your designs and use the components and styles provided by the design system effectively.
+Figma is an incredibly versatile tool, but this versatility can sometimes backfire leading to disorganization, inconsistency, and difficulty in handoffs and collaboration. These best practices are geared towards using Helios components and foundations effectively to alleviate some of these challenges. For documentation around broader Figma best practices within HashiCorp design teams, reach out to the HashiCorp [Design Ops team](https://sites.google.com/hashicorp.com/designknowledgehub/design-system) or visit the [#design-ops slack channel](https://hashicorp.slack.com/archives/C029GL8GJDV).
 
-!!! Info
+## Component availability
 
-These best practices have been curated by the Design Systems Team, Figma has its own set of [best practices](https://www.figma.com/best-practices/) for using the tool effectively.
+**Structure** is the legacy set of components and foundations used to connect HashiCorp products and consists Ember components and foundational styles paired with a limited set of Figma counterparts. **Structure is no longer supported and is in the process of deprecation and sunsetting.**
```

Comment:
I liked this in the banner, actually. I don't think the details about what is included in Structure (ie. Ember components, foundational styles, Figma counterparts, etc) are necessary since it is being sunset...feels a little too verbose. Structure also wasn't widely used, so I recommend adjusting some of the language for accuracy. 

```suggestion
**What is Structure?**

Structure is the legacy Design System used primarily by Cloud UI. **Structure is no longer supported and we're in the process of sunsetting it.**
```